site/src/components/head/Stylesheet.astro

```diff
@@ -0,0 +1,15 @@
+---
+import { getVersionedBsCssProps } from '@libs/bootstrap'
+import type { Layout } from '@libs/layout'
+
+interface Props {
+  direction?: 'rtl'
+  layout: Layout
+}
+
+const { direction } = Astro.props
+---
+
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@docsearch/css@3" />
```

Comment:
This shouldn't be loaded from jsDelivr IMHO. It should be loaded from us like we did before. All assets should come from us with the exception of maybe examples.

Reply:
I've probably missed a commit while rebasing everything. Thanks for the comment, I'm checking it 👌

Reply:
Fortunately, it was handled correctly. Just forgot to remove this line. It's now done via https://github.com/twbs/bootstrap/pull/41251/commits/9b912486fe20747924457d684343ed802896652c. Good catch :)